modules/svg/points.js

```diff
@@ -122,6 +123,22 @@ export function svgPoints(projection, context) {
             .append('path')
             .call(markerPath, 'shadow');
 
+        // add a ring if there's a radius/diameter tag
+        enter
+            .filter((d) => getRadiusTag(d.tags))
+            .append('circle')
+            .attr('class', 'radius')
+            .attr('data-nodeId', node => node.id);
```

Comment:
it's probably better to avoid camelcasing in html attributes, see e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/data-*

```suggestion
            .attr('data-nodeid', node => node.id);
```

or maybe even simpler:

```suggestion
            .attr('data-id', node => node.id);
```